Tests/SwiftGenKitTests/JSONTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+// SwiftGenKit UnitTests
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import PathKit
+@testable import SwiftGenKit
+import XCTest
+
+class JSONTests: XCTestCase {
+  func testEmpty() throws {
+    let parser = try JSON.Parser()
+
+    let result = parser.stencilContext()
+    XCTDiffContexts(result, expected: "empty", sub: .json)
+  }
+
+  func testDictionary() throws {
+    let parser = try JSON.Parser()
+    try parser.searchAndParse(path: Fixtures.path(for: "configuration.json", sub: .json))
+
+    let result = parser.stencilContext()
+    XCTDiffContexts(result, expected: "configuration", sub: .json)
+  }
+
+  func testArray() throws {
+    let parser = try JSON.Parser()
+    try parser.searchAndParse(path: Fixtures.path(for: "array.json", sub: .json))
+
+    let result = parser.stencilContext()
+    XCTDiffContexts(result, expected: "array", sub: .json)
+  }
+
```

Comment:
We're missing the `testDirectoryInput` here (same as in YAML tests), which produces a `all.yaml` context.